bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -70,7 +76,7 @@ object GemsStrategy extends AgsStrategy {
 
   // Convert from catalog results to GeMS-specific results.
   private def toGemsCatalogSearchResults(ctx: ObsContext, futureAgsCatalogResults: Future[List[CatalogResultWithKey]]): Future[List[GemsCatalogSearchResults]] = {
-    val anglesToTry = (0 until 360 by 45).map(x => new Angle(x, Angle.Unit.DEGREES))
+    val anglesToTry = (0 until 360 by 45).map(x => Angle.fromDegrees(x))
```

Comment:
Another trivial point but I think the η-reduction makes it read a bit better:  `(0 until 360 by 45).map(Angle.fromDegrees)`